docs/source/asr/models.rst

```diff
@@ -148,6 +148,9 @@ the transducer config at ``<NeMo_git_root>/examples/asr/conf/fastconformer/fast-
 
 Note that both configs are subword-based (BPE).
 
+You can also train these models with longformer-style attention using the following configs: CTC config at
```

Comment:
Would you please add more detail on the benefits and limitations of this configurations compared to the regular one?
Also a reference to the Lonformer paper would help.

Reply:
Added reference and more explanation